lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -824,8 +826,11 @@ function dosomething_campaign_is_active($node) {
   */
  function dosomething_campaign_is_win_module($vars) {
    $today = new DateTime();
+   $today->setTimezone(new DateTimeZone('America/New_York'));
    $end_date = new DateTime($vars['field_high_season'][0]['value2']);
+   $end_date->setTime(23, 59, 59);
    $win_module_end = date_modify(clone $end_date, '+30 days');
+   $win_module_end->setTime(23, 59, 59);
```

Comment:
might make sense to abstract this to a `dosomething_campaign_set_est_timezone` function instead of duplicating the code twice.